vendor/bats-core/lib/bats-core/tracing.bash

```diff
@@ -120,7 +120,7 @@ bats_trim_filename() {
 bats_debug_trap() {
 	# don't update the trace within library functions or we get backtraces from inside traps
 	if [[ "$1" != $BATS_ROOT/lib/* && "$1" != $BATS_ROOT/libexec/* ]]; then
-		# The last entry in the stack trace is not useful when en error occured:
+		# The last entry in the stack trace is not useful when en error occurred:
```

Comment:
@sobolevn presumably this change should get submitted to bats-core, too

Reply:
Done already! 🙂